ros/src/styx/bridge.py

```diff
@@ -137,7 +137,7 @@ def publish_odometry(self, data):
         self.broadcast_transform("base_link", position, orientation)
 
         self.publishers['current_pose'].publish(pose)
-        self.vel = data['velocity']* 0.44704
+        self.vel = data['velocity'] * 0.2
```

Comment:
Such magic numbers should be declared in a constant lists and/or as ROS parameters.

Reply:
moved it into a constant variable